### PR TITLE
chore(gh actions): lock closed issues after inactivity

### PR DESCRIPTION
**Pull request by siegerts**
#### Description of changes

- chore(issue picker): add link to Discord server as an option for help
- chore(gh actions): add action to locked closed threads after 30 days of inactivity

#### Checklist

- [x] PR description included

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by siegerts** [comment]
This will run every hour since only 50 issues and pull requests will be handled at once. After that's processed, it schedule can be reduced.

**Review by cjihrig — COMMENTED** [review]
This LGTM, thank you. Maybe we should consider forking https://github.com/dessant/lock-threads so we have full control over things.

I'm not going to "Approve" the PR yet. I want to make sure it doesn't accidentally get merged before others have a chance to weigh in.

**Review by attilah — APPROVED** [review]
LGTM, I like the idea of this kind of closure for issues and PRs.

File: .github/workflows/lock.yaml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+name: 'Lock Threads'
+
+on:
+  schedule:
+    - cron: '0 * * * *'
```
Comment (kaustavghosh06) [inline .github/workflows/lock.yaml:5]: Does this cron run daily?

File: .github/workflows/lock.yaml [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+jobs:
+  lock:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: dessant/lock-threads@v2
```
Comment (kaustavghosh06) [inline .github/workflows/lock.yaml:11]: Is this a reliable and widely used plugin which we can trust and use?

File: .github/ISSUE_TEMPLATE/config.yml [hunk 3]
```diff
@@ -0,0 +1,5 @@
+blank_issues_enabled: true
```
Comment (kaustavghosh06) [inline .github/ISSUE_TEMPLATE/config.yml:1]: Where would this issue template be used and is this relevant to this PR?

Comment (siegerts) [reply .github/ISSUE_TEMPLATE/config.yml:1]: It persists the current functionality (image below ) while also adding a link to Discord server as an option for help

image

File: .github/workflows/lock.yaml (hunk 1 above)
Comment (siegerts) [reply .github/workflows/lock.yaml:5]: This will run every hour since only 50 issues and pull requests will be handled at once. After that's processed, the schedule can be reduced.

File: .github/workflows/lock.yaml (hunk 2 above)
Comment (siegerts) [reply .github/workflows/lock.yaml:11]: Used in below repos - 
- [Jest](https://github.com/facebook/jest/blob/master/.github/workflows/lock.yml) 
- [Flutter ](https://github.com/flutter/flutter/blob/master/.github/lock.yml)
- [Next.js](https://github.com/vercel/next.js/blob/canary/.github/lock.yml)

**Review by kaustavghosh06 — APPROVED** [review]
